mysql/tram-saga-schema.sql

```diff
@@ -0,0 +1,49 @@
+USE eventuate;
+
+DROP Table IF Exists aggregate_instance_subscriptions;
```

Comment:
The base image is `eventuateio/eventuate-tram-mysql:0.3.0.RELEASE`. It should contain this schema.

Reply:
I will check